bot/cogs/help.py

```diff
@@ -68,13 +69,16 @@ class HelpSession:
         Where the help message is to be sent to.
     """
 
-    def __init__(self, ctx, *command, cleanup=False, only_can_run=True, show_hidden=False, max_lines=15):
+    def __init__(
+        self, ctx: Context, *command, cleanup: bool = False, only_can_run: bool = True,
+        show_hidden: bool = False, max_lines: int = 15
+    ):
         """
         Creates an instance of the HelpSession class.
 
         Parameters
         ----------
-        ctx: :class:`discord.Context`
+        ctx: :class:`discord.ext.commands.Context`
```

Comment:
I thought we were doing away with this sort of format in the docstrings. It's also present in class docstrings here but I cannot leave separate review comments for them.